docker.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import sys 
+import subprocess
+import os
+
+pythonVersion = '3.5'
+ptkVersion = '1.00'
+
+if len(sys.argv) > 1:
+    pythonVersion = sys.argv[1]
```

Comment:
Can you use argparse here? This seems prone to errors if someone flip flops the arguments